java/ql/lib/ext/java.security.cert.model.yml

```diff
@@ -0,0 +1,6 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["java.security.cert", "X509CertSelector", False, "setSubjectPublicKey", "", "", "Argument[0]", "credentials-key", "hq-generated"]
```

Comment:
Should this model and any others with an empty signature string be switched back to using their original explicit signature string (like you did for [these models](https://github.com/github/codeql/pull/13978#discussion_r1311891094) and [these models](https://github.com/github/codeql/pull/13978#discussion_r1311910926))? 
I'm wondering if any of these cases are creating extra models if there are more overloads for the method than were originally modeled in SensitiveApi.qll.

Reply:
I've gone back and converted the models with multiple overloads for a method to explicitly list them.

> I'm wondering if any of these cases are creating extra models if there are more overloads for the method than were originally modeled in SensitiveApi.qll.

I did find one instance of that, but it was in the `net.schmizz.sshj` models.